interface/stylesheet.dm

```diff
@@ -66,6 +66,12 @@ h1.alert, h2.alert		{color: #000000;}
 .suicide				{color: #ff5050;	font-style: italic;}
 .green					{color: #03ff39;}
 .shadowling				{color: #3b2769;}
+.velvet					{color: #21007F;}
+.velvet_bold			{color: #21007F; font-weight:bold;}
```

Comment:
basically what doing this does is it uses all applicable span classes; so doing velvet bold applies both the 'velvet' and 'bold' span classes.